docs/subnets/create-a-vm-blobvm.md

```diff
@@ -0,0 +1,1672 @@
+# Create a Blob Virtual Machine (VM)
+
+## Introduction
+
+Virtual Machine is a blueprint for a blockchain and a blockchain instantiates from VM, very much similar to a class-object relationship. Issuance of transactions, transaction types, block structure, block building algorithm, keeping pending transactions in the mempool, gossiping transactions to the connected nodes, etc. are defined in a virtual machine.
+
+Virtual Machines can be broken down into components like **Transaction**, **Mempool**, **Network**, **Block**, etc. Each component works together to:
```

Comment:
Avoid using "etc", since this is a tutorial we shouldn't presume knowledge from the reader. Also I don't think a VM's interface requires us to implement a mempool (even though it is a common abstraction used).

Reply:
"etc" will be explained later in the tutorial